design/FY2019/Interzone Air Wall NFP and Design.md

```diff
@@ -0,0 +1,192 @@
+objects the same as always. For simulations using AirflowNetwork, it gets a little more complicated. For smaller openings,
+such as open doorways, the current `AirflowNetwork:MultiZone:Component:DetailedOpening`, 
+`AirflowNetwork:MultiZone:Component:SimpleOpening` and `AirflowNetwork:MultiZone:Component:HorizontalOpening`
+objects should be adequate. For very large openings (horizontal and vertical) another approach will be required. One
+possibility is to use a superzone for the AirflowNetwork calculations. Or perhaps a new opening type could be developed
+to approximate the connection.
+
+#### Pitfalls ####
+ - It's unclear what the values should be for the simple mixing models across an open boundary.
+ - AirflowNetwork reporting may be confusing if a superzone approach is used.
+ 
+## Testing/Validation/Data Sources ##
+
+Energy balances and comparisons with simlations using detailed zoning and combined zones will be used to confirm
+that the results are reasonable. Various levels of reporing including design component loads will all need to
+be consistent with each other.
+
+## Input Output Reference Documentation ##
+
+The proposed input mechanism is to modify the `BuildingSurface:Detailed` and `FenestrationSurface:Detailed` objects to allow a new
+choice of *AirWall* for Surface Type. When this is selected, the Construction input is left blank (or ignored with a warning).
+ Also, the only allowed choices for Outside Boundary Condition will be *Surface* and *Zone*.
+
+## Input Description ##
+
+**BuildingSurface:Detailed**
+
+Field: Surface Type
+
+. . . *AirWall* indicates an open boundary between two zones. When this option is selected, Construction Name is ignored and the
+Outside Boundary Condition must be either *Surface* or *Zone*.
+
+Field: Construction Name
+
+. . . This field is ignored if the Surface Type is *AirWall*.
+
+Field: Outside Boundary Condition
+
+. . . If Surface Type is *AirWall* the only valid options are *Surface* and *Zone*.
+
+**FenestrationSurface:Detailed**
+
+Field: Surface Type
+
+. . . *AirWall* indicates an open boundary between two zones. When this option is selected, Construction Name is ignored and the
+Outside Boundary Condition must be either *Surface* or *Zone*.
+
+Field: Construction Name
+
+. . . This field is ignored if the Surface Type is *AirWall*.
+
+
+## Outputs Description ##
+
+Various output descriptions may need notes to explain what happens when energy crosses an airwall boundary.
+
+## Engineering Reference ##
+
+TBD
+
+## Example File and Transition Changes ##
+
+One or more example files will be developed which use airwalls in various ways.
+
+## References ##
+
+None.
+
+## Design ##
+
+The overall plan is to tackle radiant exchange first, then solar/daylighting, then air exchange.
+
+### Surface Class ###
+Anywhere that `Surface(n).Class` is used will be evaluated to add a case or if block for 
+the new `SurfaceClass_AirWall`.
+
+### SuperZones ###
```

Comment:
Is it possible to not have to explicitly list super zones?  You should be able to determine the adjacency matrix for zones connected by air walls using [this algorithm](https://stackoverflow.com/questions/32164012/how-to-get-distance-matrix-from-adjacency-matrix-matlab).

Reply:
Maybe I misread this, it sounds like a user does not need to explicitly enter in a SuperZone (there is no new IDD object)?  That would be good.  The only downside I can see is in reporting.  For that you could have a SuperZone report that lists the clustered SuperZones, variables could be reported for SuperZones.

Reply:
I actually wonder if the concept of SuperZones could be extended or used for other purposes.  Specifically, could they be used to partition the building into mostly disconnected pieces that could be simulated in parallel.

Reply:
> 
> 
> Is it possible to not have to explicitly list super zones? You should be able to determine the adjacency matrix for zones connected by air walls using [this algorithm](https://stackoverflow.com/questions/32164012/how-to-get-distance-matrix-from-adjacency-matrix-matlab).

There are use cases where EnergyPlus doesn't always have a full, self-contained 3D geometry. We do this for some of our residential work. I believe Title 24 for commercial buildings does as well? So I'm in favor of the explicit list.

Reply:
Should you be using airwalls if you don't have fully enclosed geometry?

Reply:
Maybe not often, but I wouldn't say never....